eventmesh-connectors/eventmesh-connector-http/src/main/java/org/apache/eventmesh/connector/http/sink/connector/HttpSinkConnector.java

```diff
@@ -120,14 +120,14 @@ private void sendMessage(ConnectRecord record) {
             .putHeader("Content-Type", "application/json; charset=utf-8")
             .sendJson(record, ar -> {
                 if (ar.succeeded()) {
+                    log.info("[HttpSinkConnector] Successfully send message via HTTP. Record: timestamp={}, offset={}", record.getTimestamp(),
+                        record.getPosition().getOffset());
                     if (ar.result().statusCode() != HttpResponseStatus.OK.code()) {
-                        log.error("[HttpSinkConnector] Failed to send message via HTTP. Response: {}", ar.result());
-                    } else {
-                        log.info("[HttpSinkConnector] Successfully send message via HTTP. ");
+                        log.error("[HttpSinkConnector] Unexpected response received. StatusCode: {}", ar.result().statusCode());
```

Comment:
The timestamp and offset are already recorded when the request is successfully made, is it necessary to repeat them here? Also, we only care about the status code of the response, do we need to print the full response?

Reply:
>The timestamp and offset are already recorded when the request is successfully made, is it necessary to repeat them here?

Yeah, it would be helpful for users to find which message failed.

>do we need to print the full response?

Usually failed messages won't be to many, so printing their full response won't flood your screen, and would be useful for users to figure out what happened.

You reminded me that it would be good to print successful deliveries' full response under **debug** log level.

Reply:
In my testing, I found that HttpResponse doesn't override the `toString()` method, so it doesn't directly print the full request, so I just logged the status code instead.

Reply:
You can serialize it with FastJSON2.

Reply:
Here are some snippets of HttpResponse and they may be helpful: `https://www.tabnine.com/code/java/classes/io.vertx.ext.web.client.HttpResponse`. For detailed usages, please refer to vertx docs.